packages/components/src/color-picker/test/index.js

```diff
@@ -43,11 +41,7 @@ function moveReactColorfulSlider( sliderElement, from, to ) {
 	fireEvent( sliderElement, new FakeMouseEvent( 'mousemove', to ) );
 }
 
-const sleep = ( ms ) => {
-	const promise = new Promise( ( resolve ) => setTimeout( resolve, ms ) );
-	jest.advanceTimersByTime( ms + 1 );
-	return promise;
-};
+const sleep = ( ms ) => new Promise( ( resolve ) => setTimeout( resolve, ms ) );
```

Comment:
This should probably get replaced with something like:
```js
await waitFor( () => expect( onChange ).toHaveBeenCalled() );
```
at all the places where `sleep()` is used.

Reply:
Thanks for the suggestion, done in https://github.com/WordPress/gutenberg/pull/47056/commits/8e64eb4489124c7d5148b45afbab648b8c346b38

Reply:
There are still the "we need to sleep for at least 1ms" comments that no longer make any sense.

Reply:
I mean, they make sense IMHO but I'm happy to remove them.

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/47056/commits/5b2ae5aaff39f98c60596f3969ecaadccfa92c5f

Reply:
You're right, now I realize the actually do make sense. I apologize for being so blunt.

Reply:
No worries. I think you have a point that the `waitFor()` calls should be self-explanatory, making the comment redundant.